browser/scripts/ui.js

```diff
@@ -196,20 +260,80 @@ VizorUI.prototype.onSearchResultsChange = function() {
 		presetsLib.height('auto');
 		presetsList.height(maxHeight);
 	}
-	this.updateState();
 };
 
 VizorUI.prototype.onBtnPresetsClicked = function() {
 	if (!this.isVisible()) return false;
-	this.state.visibility.panel_presets = !this.state.visibility.panel_presets;
-	this.applyVisibility();
+	var v = this.state.visibility;
+	if (!v.floating_panels) {
+		// nothing displayed so we expect just the presets
+		v._.panel_chat = false
```

Comment:
because this will modify visibility.floating_panels on the assumption that the panels are off but we expect to see a panel, so that will generate the event to panel_chat anyway.